### PR TITLE
Upgrade Dependencies

### LINKED ISSUE #20: API server start fails when using Fn::GetAtt when Resources are empty
**Describe the bug**
I'm passing the AppSync GraphQL API endpoint to functions via intrinsic function in environment variables. It works with no problem when deployed. But when I try running it locally, I get error:
```
AppSync Simulator: TypeError: Cannot read property 'GraphQlApi' of undefined
```

**To Reproduce**
Put into `provider.environment` for examlpe:
```yaml
API_URL: { 'Fn::GetAtt': ['GraphQlApi', 'GraphQLUrl'] }
```
and run `$ serverless offline start`

**Expected behavior**
There should be no error when starting the API server and the value should be taken from `custom.appsync-simulator.getAttMap`.

**Screenshots**
image

**Additional context**
I figured out that this issue comes from the `cfn-resolver-lib`.
To be exact - from [ResourcesNode.findWrappedResource](https://github.com/robessog/cfn-resolver-lib/blob/master/src/nodeTypes/ResourcesNode.js#L10)

The error disappears when I put some contents into the Resources for example like this:
```yaml
resources:
  Resources:
    abc: def
```
it causes, that `this.wrappedObject` is not `undefined` anymore because [`addChild`](https://github.com/robessog/cfn-resolver-lib/blob/master/src/nodeTypes/ObjectNode.js#L10-L13) method was called on [`ResourcesNode` ](https://github.com/robessog/cfn-resolver-lib/blob/master/src/nodeTypes/ResourcesNode.js) and thus `this.wrappedObj = {}` was performed.

Possible quick solution that comes to my mind is doing something like this in [index.js](https://github.com/bboure/serverless-appsync-simulator/blob/master/src/index.js#L155):
```diff
- Resources: this.serverless.service.resources?.Resources || {},
+ Resources: this.serverless.service.resources?.Resources || { a: 'b' }, 
```
But that's quite ugly solution and I don't really like it.

Any ideas how to properly fix it as easily as possible?

### PR DESCRIPTION
**Pull request by bboure**
Fixes #20 

Blocked by https://github.com/aws-amplify/amplify-cli/pull/4285

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,21 +19,22 @@
...
-    "amplify-util-mock": "^3.18.0",
-    "aws-sdk": "^2.659.0",
+    "amplify-appsync-simulator": "^1.20.2",
+    "amplify-nodejs-function-runtime-provider": "^1.0.4",
+    "aws-sdk": "^2.678.0",
     "axios": "^0.19.2",
-    "cfn-resolver-lib": "^1.1.5",
+    "cfn-resolver-lib": "^1.1.6",
     "dataloader": "^2.0.0",
+    "extend": "^3.0.2",
```
Comment (bboure) [inline package.json:28]: watch https://github.com/aws-amplify/amplify-cli/pull/4285
when merged, can be removed
otherwise, this needs to be included

**Comment by FilipPyrek** [comment]
@bboure could you please for now do the merge and publish with the `extend` package included?
Because we really need that bugfix of `cfn-resolver-lib` to be released.

**Comment by bboure** [comment]
@FilipPyrek Yeah, I guess it could take some time for them to merge and release. 
I also have a PR open waiting for approval.
I'll merge and release a new version.

**Comment by bboure** [comment]
Released in v0.5.0 🚀

**Pull request merged**